api/src/routes/course-route.js

```diff
@@ -199,7 +199,7 @@ router.delete("/:courseCode", async (req, res) => {
                 LOGGER.error("Course does not exist");
                 res.status(404).json({ msg: "Requested object not found" });
             } else {
-                LOGGER.info("DELETE request succeeded for /Course/${req.params.courseCode}");
+                LOGGER.info(`DELETE request succeeded for /Course/${req.params.courseCode}`);
```

Comment:
Do we have a test to check the correct message is printed?

Reply:
Might also need to change plan-route.js cause i mightve copied it from here

Reply:
uh how do u test logger statements?
I think it should be fine cause I remember I changed the other files maybe I just missed this one

Reply:
> uh how do u test logger statements?
> I think it should be fine cause I remember I changed the other files maybe I just missed this one

O nvm thought it was a response status. Don't think u can test loggers.
Yea plan-route.js is broke